spec/20-http_request_header_format.md

```diff
@@ -352,6 +352,6 @@ Vendors receiving a `tracestate` request header MUST send it to outgoing request
 
 Following are allowed mutations:
 
-- **Update key value**. The value of any key can be updated. Modified keys MUST be moved to the beginning (left) of the list. This is the most common mutation resuming the trace.
-- **Add a new key/value pair**. The new key-value pair SHOULD be added to the beginning of the list.
+- **Add a new key/value pair**. The new key/value pair SHOULD be added to the beginning of the list.
```

Comment:
You changed the meaning of this one.

```suggestion
- **Add a new key/value pair**. The new key/value pair MUST be added to the beginning of the list.
```

Reply:
No I just reordered the items to create->update->delete since creating has to come first before one can update a value.

Reply:
Ah I see. It is confusing that an updated key MUST be moved to the left, but a newly added key only SHOULD be added on the left. If you delete your key, but do not follow the optional (SHOULD) for re-adding a new key, haven't you made a spec-noncompliant edit? I think this is a contradiction in the spec.

In any case, it is fine here as that is not the issue.